proof.go

```diff
@@ -88,9 +88,9 @@ func (pin proofInnerNode) Hash(childHash []byte) []byte {
 //----------------------------------------
 
 type proofLeafNode struct {
-	Key       cmn.HexBytes `json:"key"`
-	ValueHash cmn.HexBytes `json:"value"`
-	Version   int64        `json:"version"`
+	Key       []byte `json:"key"`
```

Comment:
are you sure this won't have any effects?

Reply:
will investigate, looked into go-amino, but couldn't find a reason for why someone would need to use cmn.hexbytes. Will dig through the SDK next

Reply:
```
// The main purpose of HexBytes is to enable HEX-encoding for json/encoding.
type HexBytes []byte
```

Reply:
its back, just from tm-cmn